test/quantization/test_quant_api.py

```diff
@@ -167,7 +154,7 @@ class TestQuantFlow(unittest.TestCase):
     def test_dynamic_quant_gpu_singleline(self):
         m = ToyLinearModel().eval()
         example_inputs = m.example_inputs()
-        m = _apply_dynamic_quant(m)
+        m = quantize(m, get_apply_int8dyn_quant())
```

Comment:
If this is going to be the top level API I'd prefer making the interface a a string here

so people can pass in `quantize(m , "int4_wo")` this mimics what we do in compile and users wont have to explicitly import each callable

Reply:
int4wo also has additional args like groupsize though?

Reply:
Ah I see ok then having the input be a callable seems fine as is